Program.cs

```diff
@@ -1,7 +1,17 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter pharse or word: ");
+         string input_word = Console.ReadLine ();
+         string word = input_word.Replace (" ", string.Empty);

```

Comment:
Replace() and String Concatenation operations could be avoided using the Two Pointer technique.

Reply:
Using two pointer technique.